lib/feature/wallet/widgets/account_asset_tab/token_wallet_asset/token_wallet_asset_widget.dart

```diff
@@ -30,6 +30,7 @@ class TokenWalletAssetWidget extends StatelessWidget {
         balanceService: inject<BalanceService>(),
         currencyConvertService: inject<CurrencyConvertService>(),
         nekotonRepository: inject<NekotonRepository>(),
+        balanceStorage: inject(),
```

Comment:
Why without type?

Reply:
Not necessary, doing it for more than 2 months))